pwndbg/commands/__init__.py

```diff
@@ -58,36 +63,16 @@ class CommandCategory(str, Enum):
     DEV = "Developer"
 
 
-def list_current_commands():
-    current_pagination = gdb.execute("show pagination", to_string=True)
-    current_pagination = current_pagination.split()[-1].rstrip(
-        "."
-    )  # Take last word and skip period
-
-    gdb.execute("set pagination off")
-    command_list = gdb.execute("help all", to_string=True).strip().split("\n")
-    existing_commands: Set[str] = set()
-    for line in command_list:
-        line = line.strip()
-        # Skip non-command entries
-        if (
-            not line
-            or line.startswith("Command class:")
-            or line.startswith("Unclassified commands")
-        ):
-            continue
-        command = line.split()[0]
-        existing_commands.add(command)
-    gdb.execute(f"set pagination {current_pagination}")  # Restore original setting
-    return existing_commands
-
-
-GDB_BUILTIN_COMMANDS = list_current_commands()
+GDB_BUILTIN_COMMANDS = pwndbg.dbg.commands()
 
 # Set in `reload` command so that we can skip double checking for registration
 # of an already existing command when re-registering GDB CLI commands
 # (there is no way to unregister a command in GDB 12.x)
-pwndbg_is_reloading = getattr(gdb, "pwndbg_is_reloading", False)
+pwndbg_is_reloading = False
```

Comment:
Shall we update the comment above to reflect lldb state somehow too?

Reply:
Reloading is something that can only be done for GDB, and that I feel will only be available under GDB for the foreseeable future. I guess we could add something like `Reloading isn't available outside of GDB, so just assume we're not reloading unless we're running under GDB`?